src/qae/initial-properties.js

```diff
@@ -172,10 +174,63 @@ const properties = {
  *      fontStyle: ["bold", "italic"],
  *      color: { color: "orangered" },
  *    }
+ *  },
+ *  bgColor: {
+ *    useExpression: true,
+ *    color: {
+ *      index: 6,
+ *      color: "#006580"
+ *    },
+ *    colorExpression: {
+ *      qStringExpression: {
+ *        qExpr: "red()"
+ *      }
+ *    }
+ *  },
+ *  bgImage: {
+ *    mode: "media",
+ *    mediaUrl: {
+ *      qStaticContentUrlDef: {
+ *        qUrl: "<path-to-image>"
+ *      }
+ *    }
  *  }
  * }
  */
 
+/**
+ * Chart background color
+ * @name BackgroundColor
+ * @type object
+ * @property {PaletteColor} color - Background color palette
+ */
+
+/**
+ * Chart background color by expression
+ * @name BackgroundColorExpression
+ * @type object
+ * @property {boolean} useExpression - Boolean to indicate if color by expression should be used
```

Comment:
Noted that setting background expression via the property panel will use `useColorExpression` while nebula only consumes `useExpression`. Most likely a bug in Sense or in Nebula.

Reply:
Fix in nebula https://github.com/qlik-oss/nebula.js/pull/1426